agent/bench-scripts/pbench-fio

```diff
@@ -755,16 +755,16 @@ function fio_run_benchmark() {
 	local count=1
 	if [ "$job_mode" = "serial" ]; then
 		# if each target is separated by a space, there will be one job for each in next for loop
-		targets=`echo $targets | sed -e s/,/" "/g`
+		targets=${targets//,/ }
 	fi
 	typeset -i ntargets=$(echo $targets | wc -w)
-	typeset -i ntesttypes=$(echo $test_types | sed -e 's/,/ /g' | wc -w)
-	typeset -i nblocksizes=$(echo $block_sizes | sed -e 's/,/ /g' | wc -w)
+	typeset -i ntesttypes=$(echo ${test_types//,/ } | wc -w)
+	typeset -i nblocksizes=$(echo ${block_sizes//,/ } | wc -w)
```

Comment:
Alternately (since you're already changing these lines with an eye toward removing subprocesses...),
```
	typeset -i ntesttypes=$(wc -w <<< ${test_types//,/ })
	typeset -i nblocksizes=$(wc -w <<<  ${block_sizes//,/ })
```
(And, of course, you can probably get rid of the `wc`, too, if you use the shell's word splitting and count the result.)

Reply:
I'll perform this is a separate commit.  Thanks!

Reply:
Addressed in PR #2539.